walkthroughs/week-5-tdd/intro/src/test/java/com/google/sps/GreeterTest.java

```diff
@@ -30,4 +30,24 @@ public void testGreeting() {
 
     Assert.assertEquals("Hello Ada", greeting);
   }
+
+  @Test
+  public void testGreetingTrimsWhitespace() {
+    Greeter greeter = new Greeter();
+
+    String greeting = greeter.greet("\t\t   Ada   ");
+
+    // Whitespace should be trimmed
+    Assert.assertEquals("Hello Ada", greeting);
+  }
+
+  @Test
+  public void testGreetingNonAlphanumeric() {
+    Greeter greeter = new Greeter();
+
+    String greeting = greeter.greet("\t@#@$@ Ada !*&");
```

Comment:
Neither of these tests actually guarantee that numbers are not trimmed. One of the tests should include numbers in the string, or a new test should be created for that case.

Reply:
Good point, I definitely forgot that numbers were included in 'alpha-numeric'. Fixed.